app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    files_name = command.split(" ")
```

Comment:
1.Split by emtry string, not by " "
2.Check that lenght is 3
3.Check that first element is "cp"
4.Unpack it into variables